packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -387,10 +392,10 @@ export default class Calendar extends React.Component {
   }
 
   getDateInView = () => {
-    const { preSelection, selected, openToDate, utcOffset } = this.props
+    const { preSelection, selected, openToDate, initialTimeZone } = this.props
```

Comment:
Are you now able to remove the `utcOffset` prop and all references to it in react-datepicker since we are now using `initialTimeZone`? I don't think `utcOffset` gets used anymore after these changes. 

The references in these tests [[1](https://github.com/cerner/terra-framework/blob/c2f9850f4cba69cab2f64bb7fc0ad2006edc1056/packages/terra-date-picker/tests/jest/DatePicker.test.jsx#L19)][[2](https://github.com/cerner/terra-framework/blob/4880772b8b92bb05f99ef3f3084a8d1e4c389bdc/packages/terra-date-time-picker/tests/jest/DateTimePicker.test.jsx#L10)] should be updated to pass down `initialTimeZone` to keep a consistent test run.

[setUTCOffset](https://github.com/cerner/terra-framework/blob/e63c8c8ba6484d5122b9fb128973ce894ed733df/packages/terra-date-picker/src/react-datepicker/date_utils.js#L111), [getUTCOffset](https://github.com/cerner/terra-framework/blob/e63c8c8ba6484d5122b9fb128973ce894ed733df/packages/terra-date-picker/src/react-datepicker/date_utils.js#L138-L140) and [isSameUtcOffset](https://github.com/cerner/terra-framework/blob/e63c8c8ba6484d5122b9fb128973ce894ed733df/packages/terra-date-picker/src/react-datepicker/date_utils.js#L235-L241) can also be removed since they are no longer being used.

Reply:
changed here 9a4bd199149690dd5d681559f8ddc0ad44307779